datamodel/BUILD

```diff
@@ -1,27 +1,47 @@
-load("//meta_package:global_variables.bzl", "PUBLIC_API_PREFIX")
+load("//meta_package:global_variables.bzl", "SHARED_LIB_SUFFIX")
+load("//meta_package:global_variables.bzl", "ENGINE_BUILD_TARGET_PREFIX")
+load("//meta_package:rules.bzl", "r_cc_library")
+load("//meta_package:rules.bzl", "r_cc_shared_library")
 
-cc_library(
-    name = "engine_datamodel_core",
+_PACKAGE_NAME = "datamodel"
+
+_DATAMODEL_INCLUDE_PATH = "include"
+
+_TARGET_PACKAGE_PREFIX = "{}_{}".format(ENGINE_BUILD_TARGET_PREFIX, _PACKAGE_NAME)
```

Comment:
Just a nitpick, it si done different from other packages, but Idon't think, that we should refactor it now. Leave as is.